internal/cmd/dev/attestgithub/attestgithub.go

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package attestgithub
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/gittuf/gittuf/internal/cmd/common"
+	"github.com/gittuf/gittuf/internal/dev"
+	"github.com/gittuf/gittuf/internal/repository"
+	"github.com/spf13/cobra"
+)
+
+type options struct {
+	signingKey    string
+	repository    string
+	pullRequestID string
+}
+
+func (o *options) AddFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVarP(
+		&o.signingKey,
+		"signing-key",
+		"k",
+		"",
+		"signing key to use for signing attestation",
+	)
+	cmd.MarkFlagRequired("signing-key") //nolint:errcheck
+
+	cmd.Flags().StringVar(
+		&o.repository,
+		"repository",
+		"",
+		"path to GitHub repository, of form {owner}/{repo}",
+	)
+	cmd.MarkFlagRequired("repository") //nolint:errcheck
+
+	cmd.Flags().StringVar(
+		&o.pullRequestID,
+		"pull-request-ID",
```

Comment:
I think this should be pull request number? Pull requests also have ID's, but they're probably not what you want users interfacing with:


```sh
$ gh pr view --repo gittuf/gittuf 341 --json id,number
{
  "id": "PR_kwDOIVE5P85skkkv",
  "number": 341
}
```

Reply:
Yes, will fix.